test/integration/Cardano/Wallet/Network/HttpBridgeSpec.hs

```diff
@@ -90,6 +101,14 @@ spec = do
                         Left (NodeUnavailable _) -> True
                         _ -> error (msg res)
             action `shouldReturn` ()
+
+
+    describe "posting signed txs" $ beforeAll startBridge $ afterAll closeBridge $ do
+        it "works for an empty tx" $ \(_, network) -> do
+            let tx = Tx [] []
+            let signed = SignedTx $ BL.toStrict $ CBOR.toLazyByteString $ encodeTx tx
+            res <- runExceptT $ postTx network signed
+            res `shouldBe` (Right ())
```

Comment:
Added a failing test
```
expected: Right ()
        but got: Left (NodeUnavailable "FailureResponse (Response {responseStatusCode = Status {statusCode = 400, statusMessage = \"Bad Request\"}, responseHeaders = fromList [(\"Content-Length\",\"36\"),(\"Content-Type\",\"text/plain\"),(\"Date\",\"Thu, 28 Mar 2019 11:04:30 GMT\")], responseHttpVersion = HTTP/1.1, responseBody = \"Invalid input format for transaction\"})")
```
Not sure if this should work, but I would at least expect another error-message.